package.json

```diff
@@ -3,40 +3,42 @@
   "version": "2.0.1",
   "license": "Apache-2.0",
   "description": "A class designed to enable easily extending the built-in javascript Error, allowing typed errors.",
-  "main": "src/typed-error.js",
-  "types": "src/typed-error.ts",
+  "main": "lib/typed-error.js",
+  "types": "lib/typed-error.d.ts",
   "scripts": {
-    "prepublish": "require-npm4-to-publish",
-    "prepublishOnly": "npm test",
+    "lint": "resin-lint --typescript src/ test/ && tslint --format stylish --project .",
+    "lintfix": "tslint --fix --format stylish --project .",
+    "build": "tsc --pretty --project ./tsconfig.dist.json",
+    "test": "mocha --require ts-node/register/type-check test/*.ts && mocha --require coffeescript/register test/*.coffee",
     "pretest": "npm run build",
-    "build": "npm run prettify && tsc",
-    "prettify": "prettier --use-tabs --single-quote --write src/*.ts",
-    "test": "mocha --compilers ts:ts-node/register && mocha --compilers coffee:coffee-script/register"
+    "prepare": "npm run build"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/resin-io/typed-error.git"
   },
   "devDependencies": {
     "@types/bluebird": "^3.5.8",
-    "@types/mocha": "^2.2.41",
-    "@types/node": "^8.0.14",
+    "@types/chai": "^4.1.2",
+    "@types/chai-as-promised": "^7.1.0",
+    "@types/mocha": "^5.0.0",
+    "@types/node": "^9.6.2",
     "bluebird": "^3.5.0",
     "chai": "^4.1.0",
     "chai-as-promised": "^7.1.1",
-    "coffee-script": "~1.12.0",
-    "mocha": "^3.4.2",
-    "prettier": "^1.5.3",
-    "require-npm4-to-publish": "^1.0.0",
-    "ts-node": "^3.2.1",
-    "typescript": "2.4.2"
+    "coffeescript": "^2.2.4",
```

Comment:
NIT: resin-lint doesn't lint coffeescript 2 correctly. It won't understand await statements for example.

Reply:
I don't think that should matter too much, coffeescript is only used in testing and nothing fancy like await are used.

Reply:
No it doesn't and cs will go away eventually :+1: